djangoproject/templates/blog/entry_archive_year.html

```diff
@@ -1,12 +1,13 @@
 {% extends "base_weblog.html" %}
+{% load i18n %}
 
-{% block title %}{{ year|date:"Y" }} | Weblog{% endblock %}
+{% block title %}{{ year|date:"YEAR_MONTH_FORMAT" }} | {% translate "Weblog" %}{% endblock %}
```

Comment:
I don't think that's the right named format to use here, I'd just keep using the year:
```suggestion
{% block title %}{{ year|date:"Y" }} | {% translate "Weblog" %}{% endblock %}
```

(also in the other two instances later in the file)